### PR TITLE
feat(images-loaded): Add images loaded row details dialog - Part 1

### PR DESCRIPTION
**Pull request by priscilawebdev**
closes https://app.asana.com/0/1182975495223914/1199597399712880

This is only the first part. 

The second part will contain:
. The dropdownMenu action (this was removed from this PR due to some issues)
. Finish the delete button (some doubts need to be clarified)
. Source Name + Tooltip with a copy to clipboard functionality (There is already an open PR https://github.com/getsentry/sentry/pull/22797)
. Add Translations + Tooltips
. Polishing 

The third part will contain:
. Remotion of the `overflow: auto` from the table and make it look better on smaller devices

image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  // }
+
+  return (
+    <React.Fragment>
+      <Header closeButton>{title ?? t('Unknown name')}</Header>
+      <Body>
+        <Content>
+          <GeneralInfo>
+            <Label>{t('Address Range')}</Label>
+            <Value>{image_addr}</Value>
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx:71]: I think some images do not have address ranges (wasm), does this account for that?

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/notAvailable.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+function NotAvailable() {
+  return <Wrapper>{'\u2014'}</Wrapper>;
+}
+
+// TODO(PRISCILA): make this a common component
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/notAvailable.tsx:8]: 👏

**Review by matejminar — COMMENTED** [review]
Is this behind a feature flag? Or will it be merged only once all parts are ready?

**Comment by priscilawebdev** [comment]
@matejminar yes the modal is under the feature flag `images-loaded-v2` https://github.com/getsentry/sentry/blob/8f11940ec04bbd6bbcb51dddb35db0376e0eeccc/src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugImage.tsx#L176 😉

**Review by matejminar — APPROVED** [review]

**Pull request merged**